enterprise/frontend/src/embedding-sdk/cli/steps/setup-metabase-instance.ts

```diff
@@ -94,6 +94,7 @@ export const setupMetabaseInstance: CliStepMethod = async state => {
 
     if (!res.ok) {
       const errorMessage = await res.text();
+      spinner.fail();
```

Comment:
Without this, the spinner gets stuck on a loading state even when the request failed.